### PR TITLE
base/kbd: Add at v2.5.1

### PR DESCRIPTION
**Pull request by der-eismann**
This adds keymaps for all locales so people can type on their native keyboards on the virtual console

### CODE REVIEW

File: base/kbd/stone.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+homepage    : https://kbd-project.org/
+upstreams   :
+    - https://mirrors.edge.kernel.org/pub/linux/utils/kbd/kbd-2.5.1.tar.xz : ccdf452387a6380973d2927363e9cbb939fa2068915a6f937ff9d24522024683
+summary     : Linux keyboard tools
+description : |
+    The kbd project contains utilities for managing Linux console (Linux console, virtual terminals, keyboard, etc.)
+license     : GPL-2.0-or-later
+builddeps   :
+    - pkgconfig(pam)
+    - binary(gzip)
```
Comment (ReillyBrogan) [inline base/kbd/stone.yml:17]: FWIW I've been moving `binary` deps above `pkgconfig` ones.

File: base/kbd/manifest.x86_64.jsonc [hunk 2]
```diff
@@ -0,0 +1,784 @@
...
+				"/usr/bin/setfont",
+				"/usr/bin/setkeycodes",
+				"/usr/bin/setleds",
+				"/usr/bin/setmetamode",
+				"/usr/bin/setvtrgb",
+				"/usr/bin/showconsolefont",
+				"/usr/bin/showkey",
+				"/usr/bin/unicode_start",
+				"/usr/bin/unicode_stop",
+				"/usr/bin/vlock",
```
Comment (ReillyBrogan) [inline base/kbd/manifest.x86_64.jsonc:39]: It looks like the vlock command requires a PAM configuration file. Use the following:
```
#%PAM-1.0
auth required pam_unix.so
account required pam_unix.so
password required pam_unix.so
session required pam_unix.so
```
See [openssh](https://github.com/snekpit/main/tree/main/console/openssh) for what to name the pam file in the repo and also where/how to install it correctly.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: base/kbd/manifest.x86_64.jsonc (hunk 2 above)
Comment (ReillyBrogan) [reply base/kbd/manifest.x86_64.jsonc:39]: So this would be `pkg/vlock.pam` and would be installed to `/usr/share/defaults/etc/pam.d/vlock`

Comment (der-eismann) [reply base/kbd/manifest.x86_64.jsonc:39]: tbh I have no idea what this does, I just wanted to get the package in for the keymaps. Thanks for the hint!

File: base/kbd/stone.yml (hunk 1 above)
Comment (der-eismann) [reply base/kbd/stone.yml:17]: Yeah, forgot to sort it

**Comment by ReillyBrogan** [comment]
LGTM now

**Pull request merged**